deploy.sh

```diff
@@ -57,3 +57,11 @@ if [ $? -ne 0 ]; then
   echo 'Problem encountered building deployment resources'
   exit
 fi
+
+#
+# If running in development mode, run the web host locally
+#
+if [ "$DEVELOPMENT" == 'true' ]; then
+  cd webhost
```

Comment:
Do we actually have to use the webhost in development mode? Could the spa files be served by some sort of a webpack-dev server, etc. Then we could start only one process in the dev mode.